pilot/cmd/pilot-agent/main.go

```diff
@@ -78,13 +78,15 @@ var (
 	loggingOptions         = log.DefaultOptions()
 	outlierLogPath         string
 
-	instanceIPVar        = env.RegisterStringVar("INSTANCE_IP", "", "")
-	podNameVar           = env.RegisterStringVar("POD_NAME", "", "")
-	podNamespaceVar      = env.RegisterStringVar("POD_NAMESPACE", "", "")
-	kubeAppProberNameVar = env.RegisterStringVar(status.KubeAppProberEnvName, "", "")
-	serviceAccountVar    = env.RegisterStringVar("SERVICE_ACCOUNT", "", "Name of service account")
-	clusterIDVar         = env.RegisterStringVar("ISTIO_META_CLUSTER_ID", "", "")
-	callCredentials      = env.RegisterBoolVar("CALL_CREDENTIALS", false, "Use JWT directly instead of MTLS")
+	instanceIPVar          = env.RegisterStringVar("INSTANCE_IP", "", "")
+	podNameVar             = env.RegisterStringVar("POD_NAME", "", "")
+	podNamespaceVar        = env.RegisterStringVar("POD_NAMESPACE", "", "")
+	kubeAppProberNameVar   = env.RegisterStringVar(status.KubeAppProberEnvName, "", "")
+	serviceAccountVar      = env.RegisterStringVar("SERVICE_ACCOUNT", "", "Name of service account")
+	clusterIDVar           = env.RegisterStringVar("ISTIO_META_CLUSTER_ID", "", "")
+	callCredentials        = env.RegisterBoolVar("CALL_CREDENTIALS", false, "Use JWT directly instead of MTLS")
+	sdsEnableTokenExchange = env.RegisterBoolVar("SDS_ENABLE_TOKEN_EXCHANGE", true, "Enable token exchange for SDS")
```

Comment:
If this PR is just a short term compromise until we have a proper API, and the CA doesn't need any changes, why do we need to make changes to the CA configuration?

Reply:
What is your suggested revision? Do you suggest removing the flag SDS_ENABLE_TOKEN_EXCHANGE?